src/fparser/two/Fortran2003.py

```diff
@@ -5396,6 +5432,7 @@ def tofortran(self, tab='', isfix=None):
             else:
                 tmp.append(item.tofortran(tab=tab+'  ', isfix=isfix))
         tmp.append(end.tofortran(tab=tab, isfix=isfix))
+        py2_encode_list_items(tmp)
```

Comment:
Add a test for this as was done for r438 and r808?

Reply:
Yes, done.